pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+    console.log("Last name:", lastNameField.value);
+    console.log("Nickname:", nicknameField.value);
+    console.log("Birth Date:", birthdateField.value);
+    console.log(
+      "Complaints: ",
+      complaintField1.value +
+        " " +
+        complaintField2.value +
+        " " +
+        complaintField3.value
+    );
+    console.log("Wants Planning?: " + planning);
+  };
+
+  return (
+    <div id="patient-info-wrapper">
+      <TextareaItem
+        title="First Name"
+        placeholder="First Name"
+        autoHeight
+        id="firstname"
+      />
+
+      <TextareaItem
+        title="Last Name"
+        placeholder="Last Name"
+        id="lastname"
+        autoHeight
+      />
+
+      <TextareaItem
+        title="Nickname"
+        placeholder="Nickname"
+        id="nickname"
+        autoHeight
+      />
+
+      <List renderHeader={() => "Sex"}>
+        {sexValues.map((i) => (
+          <RadioItem
+            data-testid={`${i.label.toLowerCase()}-button`}
+            id={`${i.label.toLowerCase()}-button`}
+            key={i.value}
+            name="sex"
+            checked={sex === i.value}
+            onClick={() => setSex(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      <WhiteSpace />
+
+      <InputItem placeholder="YYYY-MM-DD" id="birthdate">
+        Birth Date
+      </InputItem>
+
+      <InputItem type="number" placeholder="Age" id="age">
+        Age{" "}
+      </InputItem>
+
+      <InputItem type="number" placeholder="Phone" id="phone">
+        Phone
+      </InputItem>
+
+      <TextareaItem title="Town" placeholder="Town" id="town" autoHeight />
+
+      <List renderHeader={() => "Visit?"}>
+        {radioYesNo.map((i) => (
+          <RadioItem
+            key={i.value}
+            data-testid={`visit-${i.label}`}
+            name="visit"
+            checked={visit === i.value}
+            onClick={() => setVisit(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      {/* Only display the 'female-only' div if the patient is female */}
+      {sex ? (
+        <div id="female-only">
+          <WhiteSpace />
+          <List renderHeader={() => "Pregnant?"}>
+            {radioYesNo.map((i) => (
+              <RadioItem
+                data-testid={`pregnant-${i.label}`}
+                key={i.value}
+                name="pregnant"
+                checked={pregnant === i.value}
+                onClick={() => setPregnant(i.value)}
+              >
+                {i.label}
+              </RadioItem>
+            ))}
+          </List>
+
+          <InputItem title="G" type="number" id="gravida">
+            G
+          </InputItem>
+
+          <InputItem title="P" type="number" id="para">
+            P
+          </InputItem>
+
+          <InputItem title="A" type="number" id="abortus">
+            A
+          </InputItem>
```

Comment:
I am unable to interact with any of these fields.

Reply:
@abpwrs, the label for the field is misleadingly large. Try clicking in the middle of the field - they can also be tabbed through. Also, it is a number so will only accept number input.

Reply:
![image](https://user-images.githubusercontent.com/33632286/79671836-2b02c980-8193-11ea-9fa0-a3d631553afe.png)

@abpwrs I can see how this tripped you up.

The input portion of the row is the same distance away from the label as all the others, but there wasn't a placeholder so you didn't notice.

@meghanmc Is there a placeholder value we could add to those fields, purely to make clear where to click for input?

Reply:
I think having a 0 as the placeholder makes sense, added in [this commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/66/commits/668419e8e551d7956be1ed58edf8826dc397dd3f)

Reply:
This is much more intuitive now. looks good.

Reply:
This is much more intuitive! Looks good.